compose/ui/ui-uikit/src/uikitMain/objc/CMPUIKitUtils/CMPUIKitUtils/CMPAccessibilityContainer.h

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2022 The Android Open Source Project
+ * Copyright 2023 The Android Open Source Project
```

Comment:
Now we can update to 2024

Reply:
No, @MatkovIvan said that Google stated that those indicate creation date and not the current one